docs/packet_structure.md

```diff
@@ -0,0 +1,50 @@
+# Packet Structure
+
+| Field         | Size (bytes)            | Description |
+|--------------|------------------------|-------------|
+| `header`     | 1                        | Contains routing type, payload type, and payload version. |
+| `payload_len` | 2                        | Length of the payload in bytes. |
+| `path_len`    | 2                        | Length of the path field in bytes. |
+| `path`        | `MAX_PATH_SIZE`          | Stores the routing path if applicable. |
+| `payload`     | `MAX_PACKET_PAYLOAD`     | The actual data being transmitted. |
```

Comment:
This is not correct. The actual over the air format is:


| Field         | Size (bytes)            | Description |
|--------------|------------------------|-------------|
| `header`     | 1                        | Contains routing type, payload type, and payload version. |
| `path_len`    | 1 (uint8)                | Length of the path field in bytes. |
| `path`        | `MAX_PATH_SIZE`          | Stores the routing path if applicable. |
| `payload`     | `MAX_PACKET_PAYLOAD`     | The actual data being transmitted. |


See packet serialization here:

https://github.com/ripplebiz/MeshCore/blob/main/src/Packet.cpp#L24-L31